src/librustc_mir/borrow_check/mod.rs

```diff
@@ -715,6 +718,17 @@ impl InitializationRequiringAction {
 }
 
 impl<'cx, 'gcx, 'tcx> MirBorrowckCtxt<'cx, 'gcx, 'tcx> {
+    /// Returns the *reason* that the region `r` contains the given point.
+    pub(crate) fn why_region_contains_point<R>(&self, r: R, p: Location) -> Option<Rc<Cause>>
+    where
+        R: ToRegionVid,
+    {
+        let inferred_values = &self.nonlexical_cause_info.as_ref()
+            .expect("region values not yet inferred")
```

Comment:
Nit: this should say "region cause information not yet populated" or something like that

Reply:
I think I had expected to put this method onto the `RegionCausalInfo`, precisely so that the "unwrap/expect" question is pushed up to the caller. This way is OK too, but I mildly prefer it to be moved.